src/openforms/payments/contrib/ogone/plugin.py

```diff
@@ -70,6 +71,7 @@ def start_payment(self, request, payment: SubmissionPayment):
 
     def handle_return(self, request, payment: SubmissionPayment):
         action = request.query_params.get(RETURN_ACTION_PARAM)
+        payment_id = request.query_params.get(PAYMENT_ID_PARAM)
```

Comment:
```suggestion
        payment_id = request.query_params.[PAYMENT_ID_PARAM]
```
maybe? if we don't get this param, something is wrong and it's probably best that this shows up in error monitoring rather than `None` being passed in `self.apply_status`?